lib/rust/ensogl/core/src/display/shape.rs

```diff
@@ -9,6 +9,7 @@ pub mod compound;
 pub mod constants;
 pub mod primitive;
 
+pub use compound::rectangle::Rectangle;
```

Comment:
Why is `Rectangle` re-exported, but `Circle` and `RoundedRectangle` are not?

Reply:
It's complex. I was trying to refactor it, but it's really complex and needs a lot of changes. This needs to be done one day. I re-exported `RoundedRectangle` and added comment about why we are not re-exporting `Circle` for now.